flask_of_oil/oauth_filter.py

```diff
@@ -153,16 +156,16 @@ def filter(self, scopes=None, claims=None):
         try:
             validated_token = self.validator.validate(token)
         except Exception:
-            abort(make_response("Server Error", 500))
```

Comment:
Only when passing just the status code to `abort` is the corresponding error actually thrown. So in order to be able to handle it using flask's error handlers we have to return just the status code here.